---
title: MSSQLSERVER_8630 | Microsoft Docs
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.component: errors-events
ms.reviewer: ''
ms.suite: sql
ms.technology: supportability
ms.tgt_pltfrm: ''
ms.topic: language-reference
helpviewer_keywords:
- 8630 (Database Engine error)
ms.assetid: 45a9d084-b5ad-435b-ac19-5f4352515817
caps.latest.revision: 14
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: e4d2c566a6d4620b53154477bffd4737658cfa11
ms.sourcegitcommit: f1caaa156db2b16e817e0a3884394e7b30fb642f
ms.translationtype: HT
ms.contentlocale: es-ES
ms.lasthandoff: 05/04/2018
---
# <a name="mssqlserver8630"></a>MSSQLSERVER_8630
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
  
## <a name="details"></a>Detalles  
  
|||  
|-|-|  
|Nombre del producto|SQL Server|  
|Identificador del evento|8630|  
|Origen del evento|MSSQLSERVER|  
|Componente|SQLEngine|  
|Nombre simbólico|EXECUTION_ERR|  
|Texto del mensaje|Error interno del procesador de consultas: error inesperado durante la ejecución.|  
  
## <a name="explanation"></a>Explicación  
Se ha producido un error interno al procesar la consulta.  
  
## <a name="user-action"></a>Acción del usuario  
Póngase en contacto con su proveedor de servicios principal. Proporcione la información de este mensaje de error.  
  
